OpenRA.Game/Server/Server.cs

```diff
@@ -982,6 +1141,54 @@ public void SyncClientPing()
 			DispatchOrders(null, 0, Order.FromTargetString("SyncClientPings", clientPings.WriteToString(), true).Serialize());
 		}
 
+		public void InitializeRecorderMetadata()
+		{
+			gameInfo = new GameInformation
+			{
+				Mod = Game.ModData.Manifest.Id,
+				Version = Game.ModData.Manifest.Metadata.Version,
+				MapUid = Map.Uid,
+				MapTitle = Map.Title,
+				StartTimeUtc = DateTime.UtcNow
+			};
+
+			recorder.Metadata = new ReplayMetadata(gameInfo);
+			foreach (var conn in Conns)
```

Comment:
Testing ingame shows that this is missing bots! Maybe loop over `LobbyInfo.Slots` instead?

Reply:
Looping over `var client in LobbyInfo.Clients` appears to work.